testsetups/keeper.go

```diff
@@ -26,7 +26,6 @@ type KeeperBlockTimeTest struct {
 
 	keeperRegistry          contracts.KeeperRegistry
 	keeperConsumerContracts []contracts.KeeperConsumerPerformance
-	mockServer              *client.MockserverClient
```

Comment:
is it moved to some other structure or not needed at all?

Reply:
Not needed, seems the test doesn't utilize it at all